nextflow.config

```diff
@@ -9,7 +9,6 @@ params {
 
   ref = 'https://github.com/czbiohub/test-datasets/raw/msspe/reference/MN908947.3.fa'
   ref_host = 'http://hgdownload.cse.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz'
-  ref_gb = "https://github.com/czbiohub/test-datasets/raw/olgabot/msspe--add-human-and-kraken/reference/MN908947.3.gb"
```

Comment:
References to `ref_gb` also need to be removed from `main.nf` (this is what is causing the CI to fail)